test/complex.cpp

```diff
@@ -43,12 +43,12 @@ constexpr auto test(complex<int> lhs, complex<long long> rhs)
 int main()
 {
     // Constructor tests
-    constexpr imaginary<int> i1;
+    constexpr imaginary<int> i1{};
```

Comment:
Those ones are not necessary anymore since I (hopefully) fixed in a commit a few minutes ago.